providers/netcore/src/lib/netcore.provider.ts

```diff
@@ -22,73 +24,58 @@ export enum NetCoreStatusEnum {
       apiKey: string;
       from: string;
     }
-  ) {}
+  ) {
+    this.axiosInstance = axios.create({
+      baseURL: this.BASE_URL,
+    });
+  }
 
   async sendMessage(
     options: IEmailOptions
   ): Promise<ISendMessageSuccessResponse> {
-    const netcoreLib = await import('pepipost/lib');
-
-    netcoreLib.Configuration.apiKey = this.config.apiKey;
-
-    const controller = netcoreLib.MailSendController;
-    const body = new netcoreLib.Send();
-
-    body.from = new netcoreLib.From();
-    body.from.email = options.from || this.config.from;
-    body.subject = options.subject;
-
-    body.content = [];
-    body.content[0] = new netcoreLib.Content();
-    body.content[0].type = netcoreLib.TypeEnum.HTML;
-    body.content[0].value = options.html;
-
-    body.personalizations = [];
-    body.personalizations[0] = new netcoreLib.Personalizations();
-    body.personalizations[0].to = options.to.map((email) => {
-      const item = new netcoreLib.EmailStruct();
-      item.email = email;
-
-      return item;
-    });
-
-    if (options.cc) {
-      body.personalizations[0].cc = options.cc.map((ccItem, index) => {
-        const email = new netcoreLib.EmailStruct();
-        email.email = ccItem;
-
-        return email;
-      });
-    }
-
-    if (options.bcc) {
-      body.personalizations[0].bcc = options.bcc.map((ccItem, index) => {
-        const email = new netcoreLib.EmailStruct();
-        email.email = ccItem;
-
-        return email;
-      });
-    }
-
-    body.personalizations[0].attachments = options.attachments?.map(
-      (attachment) => {
-        const attachmentPayload = new netcoreLib.Attachments();
-        attachmentPayload.content = attachment.file.toString('base64');
-        attachmentPayload.filename = attachment.name;
-
-        return attachment;
-      }
-    );
+    const emailOptions = {
+      method: 'POST',
+      headers: {
+        api_key: this.config.apiKey,
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+      data: {
+        from: { email: options.from || this.config.from, name: 'Lesley Knope' },
+        reply_to: options.replyTo,
+        subject: options.subject,
+        content: [
+          {
+            type: 'html',
+            value: options.html,
+          },
+        ],
+        personalizations: [
+          {
+            to: options.to.map((email) => ({ email })),
+            cc: options.cc.map((email) => ({ email })),
+            bcc: options.bcc.map((email) => ({ email })),
+            attachments: options.attachments?.map((attachment) => {
+              return {
+                name: attachment.name,
+                content: attachment.file.toString('base64'),
+              };
+            }),
+          },
+        ],
+      },
+    };
 
-    const response = await controller.createGeneratethemailsendrequest(body);
+    const response = await this.axiosInstance.post('/mail/send', emailOptions);
 
     return {
```

Comment:
Are you sure that the response here is correct? 

Did you had a chance to test it out with the real provider?

Reply:
Hi @scopsy,

I did not have a chance to test it out with the real provider, since netcore is not currently accepting new signups for email api.

But I took the reference from the documentation:

[EmailApi](https://cpaasdocs.netcorecloud.com/docs/pepipost-api/c2d895762f832-send-an-email)

```json
{
  "data": {
    "message_id": "fa6cb2977cdfd457b3ac98be710ad763"
  },
  "message": "OK",
  "status": "success"
}
```